errors.go

```diff
@@ -2,6 +2,29 @@ package fargo
 
 // MIT Licensed (see README.md) - Copyright (c) 2013 Hudl <@Hudl>
 
+import (
+	"fmt"
+)
+
+type unsuccessfulHTTPResponse struct {
+	statusCode    int
```

Comment:
I would unify order with string output. Maybe message prefix first and rcode second?